pine-a64-plus/debian.yaml

```diff
@@ -0,0 +1,96 @@
+
```

Comment:
can you squash the two commits ? makes review easier

Reply:
done